src/snowcli/cli/common/cli_global_context.py

```diff
@@ -62,6 +64,12 @@ def schema(self) -> Optional[str]:
         return self._schema
 
     def set_schema(self, value: Optional[str]):
+        if (
+            value
+            and not re.compile("('|\").*('|\")$").match(value)
+            and re.compile(".+\..+").match(value)
```

Comment:
Can you add comments for those patterns?

Reply:
First is replaced and second has comment